palette/Cargo.toml

```diff
@@ -7,12 +7,16 @@ description = "Makes linear color calculations and conversion easy and accessibl
 documentation = "https://docs.rs/palette/0.5.0/palette/"
 repository = "https://github.com/Ogeon/palette"
 readme = "README.md"
-keywords = ["color", "colour", "space", "linear"]
+keywords = ["color", "conversion", "linear", "pixel", "rgb"]
 license = "MIT OR Apache-2.0"
 edition = "2018"
+categories = ["graphics", "multimedia::images", "no-std"]
 
 build = "build/main.rs"
 
+[badges]
+travis-ci = { repository = "Ogeon/palette" }
```

Comment:
Considering it doesn't show the CI state of the released code (which should always pass), I'm not sure how useful this badge is, to be honest. I have always wondered. I find most status badges to be odd, really, for this reason. The maintenance badge has the reverse problem of not being live, while the true maintenance status of the lib may change from one day to another. :man_shrugging: Not sure if I'm misunderstanding something, though...

Reply:
I agree about the badges and find them mostly to be noise. Good point about not showing the released version's build, which should always pass, so this shouldn't be necessary. I don't think much of a passing badge but a failing badge can be concerning even if it's not reason to be. 

I can strike this out. This was only included because some other projects use it and makes it show up as a badge on the search next to the version number.

Reply:
They can be fun, but I would rather skip the badges for now, unless they are directly related to the published versions. :slightly_smiling_face: 

The ones in the readme are a whole other business, but there I'm leaning more and more towards the idea of having different readmes for the repo and published crate. That's a separate topic.